CMakeLists.txt

```diff
@@ -248,9 +254,9 @@ else()
                 )
         endfunction(add_feature_target)
 
-        add_feature_target(features --format progress)
-        add_feature_target(features-pretty --format pretty)
-        add_feature_target(features-wip --format pretty --tags @wip)
+        add_feature_target(features         ${CUKE_E2E_TAGS} --format progress)
```

Comment:
Just add it before `${ARGN}` in the `add_feature_target` function instead. That reduces repetition.